### PR TITLE
Added information about configuring faster storage for etcd

### PR DESCRIPTION
**Pull request by ahardin-rh**
@sjug PTAL at this in context of the existing installation/customizing install-config.yaml content in place for 4.1 docs.

### CODE REVIEW

**Comment by sjug** [comment]
/lgtm 

Subtle but valid.

**Comment by ahardin-rh** [comment]
@mffiedler Can you please provide a QE review?
@kalexand-rh Since we discussed this a little last week, can you please provide a peer review?

**Review by kalexand-rh — COMMENTED** [review]
LGTM

File: modules/installation-aws-config-yaml.adoc [hunk 1]
```diff
@@ -62,4 +66,6 @@ sshKey: ssh-ed25519 AAAA... <3>
 <2> If you do not provide these parameters and values, the installation program
 provides the default value.
 <3> You can optionally provide the `sshKey` value that you use to access the
-machines in your cluster.
\ No newline at end of file
+machines in your cluster.
+<4> To configure faster storage for etcd, especially for larger clusters, set the
+storage type as `io1` and set `iops size` to least `500`.
```
Comment (mffiedler) [inline modules/installation-aws-config-yaml.adoc]: iops and size are separate fields - iops=speed in I/O operations/second and size=how big disk is (Gb).   I suggest "To configure faster storage for etcd, especially for larger clusters, set the storage type as `io1` and set `iops` to 4000."

**Review by mffiedler — CHANGES_REQUESTED** [review]

**Comment by mffiedler** [comment]
@ahardin-rh @sjug Where did 4000 come from for iops?  In recent master-vert tests, 2000 iops would be more than sufficient.     Not opposed to using  it as an example, but a 500Gb 4000 iops disk on AWS would be pretty expensive.

**Comment by sjug** [comment]
Thanks @mffiedler, that's from me, pulled it from the [installer docs]Vhttps://github.com/openshift/installer/blob/master/docs/user/aws/customization.md).

**Comment by mffiedler** [comment]
Ok,  it might sound nitpicky, but (because I am cheapskate) I would recommend changing the example to something more modest.   Those 5 c5.9xlarge instances with that disk would be $5K/month.    I suggest changing the instance size to c5.4xlarge and iops to 2000, if there are no objections.

**Comment by sjug** [comment]
Sounds good to me

**Comment by ahardin-rh** [comment]
@sjug @mffiedler Thanks! Updated. Merging! :rocket:

**Pull request merged**